workflow/scores/precision_recall.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import division
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment
+
+from rampwf.score_types.base import BaseScoreType
+
+from ..iou import cc_iou as iou
+
+
+def _match_tuples(y_true, y_pred):
+    """
+    Given set of true and predicted (x, y, r) tuples, determine the best
+    possible match.
+
+    Parameters
+    ----------
+    y_true, y_pred : list of tuples
+
+    Returns
+    -------
+    (idxs_true, idxs_pred, ious)
+        idxs_true, idxs_pred : indices into y_true and y_pred of matches
+        ious : corresponding IOU value of each match
+
+        The length of the 3 arrays is identical and the minimum of the length
+        of y_true and y_pred
+
+    """
+    n = len(y_true)
+    m = len(y_pred)
+
+    iou_matrix = np.empty((n, m))
+
+    for i in range(n):
+        for j in range(m):
+            iou_matrix[i, j] = iou(y_true[i], y_pred[j])
+
+    idxs_true, idxs_pred = linear_sum_assignment(1 - iou_matrix)
+
+    if (not idxs_true.size) or (not idxs_pred.size):
+        ious = np.array([])
+    else:
+        ious = iou_matrix[idxs_true, idxs_pred]
+    return idxs_true, idxs_pred, ious
+
+
```

Comment:
I'm wondering if we couldn't use an extension of this for combining predictions. Remember, the specs are:

1. it gets a list of y_preds (list of list of disks),
2. it finds matching groups (each group contains at most one disk of each y_pred)
3. it makes them "vote": only those disks get outputted which contain a member disk from at least half of the y_preds
4. it averages x, y, and r in each group.

Reply:
yes, I think we can reuse this code for the prediction class (the above indeed seems a sensible way to combine predictions)